docs/system-settings/the-system-menu.md

```diff
@@ -63,9 +63,6 @@ The Wipe menu erases the current show. This is the same as the \[New
 Show\] option on the Disk menu, but is provided here for old fashioned
 Avolites users who are used to it being on the System menu.
 
-The \[Wipe Visualiser Patch\] option will erase all fixtures from the
-visualiser without affecting the current show.
-
```

Comment:
Apologies if this has been mentioned elsewhere, just checking this needs to be removed

Reply:
@icke-siegen just following up on this

Reply:
This option simply doesn't exist. So yes, should be removed.

![image](https://user-images.githubusercontent.com/70575174/94988808-452d3a00-0570-11eb-9c4e-3e137c46523f.png)